packages/gatsby-theme-carbon/src/components/ResourceCard/resource-card-group.scss

```diff
@@ -1,4 +1,11 @@
 // Card groups
+
+// #906: We need to use the attribute selector here b.c. we don't control the column,
+// it's provided by the developers through MDX and we don't want to force an extra prop
+.resource-card-group [class*="--column"] {
```

Comment:
You can avoid the wildcard with this:
```
.resource-card-group .#{$prefix}--col-lg-4 {
  margin-bottom: 0;
}
```

Reply:
I think the `--column` approach is marginally better since we provide that class (as opposed to Carbon), and if we ever let in column props, we can't guarantee it'll be 4 columns but we can guarantee it'll have the `--column` class on it.